### PR TITLE
Win msys2

### PR DESCRIPTION
**Pull request by hainm**
- try very minimal tests (import parmed and numpy)
- got ImportError with `pwd`, so I set it as None :D (wait for your feedback)

log: https://ci.appveyor.com/project/hainm/parmed-9qurq/build/1.0.6

### CODE REVIEW

File: parmed/gromacs/gromacstop.py [hunk 1]
```diff
@@ -10,6 +10,10 @@
 from datetime import datetime
 import math
 import os
+try:
+    import pwd
+except ImportError:
+    pwd = None
```
Comment (swails) [inline parmed/gromacs/gromacstop.py]: What purpose does this serve?

Comment (hainm) [inline parmed/gromacs/gromacstop.py]: if not, `python setup.py install` will fail with ImportError for `pwd`. (it's dummy fix. lol).

Comment (swails) [inline parmed/gromacs/gromacstop.py]: But why?  That doesn't make any sense...  `pwd` is imported below and `ImportError` is caught.

Should the stuff that comes after `pwd` is imported be relegated to the `else` block of the try-except?

Comment (hainm) [inline parmed/gromacs/gromacstop.py]: man, that `pwd` stuff was fixed a while ago and I just fork my repo, not `ParmEd/ParmEd` repo.

**Pull request closed without merging**